Modules/System/Base64 Convert/src/Base64Convert.Codeunit.al

```diff
@@ -24,6 +24,29 @@ codeunit 4110 "Base64 Convert"
         exit(Base64ConvertImpl.ToBase64(String));
     end;
 
+    /// <summary>
+    /// Converts the value of the input string to its equivalent string representation that is encoded with base-64 digits.
+    /// </summary>
+    /// <param name="String">The string to convert.</param>
+    /// <param name="TextEncoding">The TextEncoding for the output string.</param>
+    /// <returns>The string representation, in base-64, of the input string.</returns>
+    procedure ToBase64(String: Text; TextEncoding: TextEncoding): Text
+    begin
+        exit(Base64ConvertImpl.ToBase64(String, TextEncoding));
+    end;
+
+    /// <summary>
+    /// Converts the value of the input string to its equivalent string representation that is encoded with base-64 digits.
+    /// </summary>
+    /// <param name="String">The string to convert.</param>
+    /// <param name="TextEncoding">The TextEncoding for the output string.</param>
+    /// <param name='Codepage">The Codepage if TextEncoding is MsDos or Windows.</param>
```

Comment:
The codepage if TextEncoding is MS-DOS or WIndows.

Reply:
Again, MsDos is how the option is called on TextEncoding.